packages/components/src/mobile/keyboard-aware-flat-list/index.android.js

```diff
@@ -2,17 +2,18 @@
  * External dependencies
  */
 import { FlatList } from 'react-native';
-
 /**
  * Internal dependencies
  */
 import KeyboardAvoidingView from '../keyboard-avoiding-view';
 
-export const KeyboardAwareFlatList = ( props ) => (
-	<KeyboardAvoidingView style={ { flex: 1 } }>
-		<FlatList { ...props } />
-	</KeyboardAvoidingView>
-);
+export const KeyboardAwareFlatList = ( props ) => {
+	return (
+		<KeyboardAvoidingView style={ { flex: 1 } }>
+			<FlatList ref={ props.listRef } { ...props } />
+		</KeyboardAvoidingView>
+	);
+};
```

Comment:
I'm curious, what was the reason for changing the lines surrounding `<FlatList ref={ props.listRef } { ...props } />`? It seems the other lines changed, but are not directly relevant to the PR. Was it due to a formatting change that the editor did on save?

Reply:
Yes, it seems the editor probably formatted this. I will revert this and I will be mindful of it in other PRs 👍

Reply:
Makes sense, I've had some formatting issues with the editor too.